core/src/rpc/clementine.rs

```diff
@@ -1507,13 +1507,20 @@ pub mod clementine_aggregator_client {
             self
         }
         /// Sets up the system of verifiers, watchtowers and operators by:
-        /// 1. Collecting verifier keys, then distributing all verifier keys to all verifiers
-        /// 2. Collecting all operator configs, and distributing them to all verifiers
-        /// 3. Collecting all possible Winternitz pubkeys (determined by operator idx, seqcol idx, kickoff idx) from watchtowers, and distributing to verifiers.
+        ///
+        /// 1. Collects verifier keys from each verifier
+        /// 2. Distributes these verifier keys to all verifiers
+        /// 3. Collects all operator configs from each operator
+        /// 4. Distributes these operator configs to all verifiers
+        /// 5. Collects all possible Winternitz pubkeys (determined by operator idx, seqcol idx, kickoff idx) from each watchtower
+        /// 6. Distributes these Winternitz pubkeys to all verifiers
         pub async fn setup(
             &mut self,
             request: impl tonic::IntoRequest<super::Empty>,
-        ) -> std::result::Result<tonic::Response<super::Empty>, tonic::Status> {
+        ) -> std::result::Result<
+            tonic::Response<super::VerifierPublicKeys>,
```

Comment:
why return the pubkeys btw?

Reply:
I guess we can keep this. This also made me think about verifier initialization for the dynamic verifier set feature. We'll need an endpoint to return the current verifier set.